src/specs/02_advanced/hybrid_validator_management_manage_besu.spec

```diff
@@ -0,0 +1,55 @@
+# QBFT consensus - Validator and non-validator behaviour
+
+  Tags: networks/typical-hybrid::hybrid-template-q2b1, pre-condition/no-record-blocknumber, hybrid-validator-management-manage-besu
+
+This specification describes how validator and non-validator nodes behave in hybrid network
+ - validator node can seal new blocks
+ - non-validator node is not authorized to seal new blocks but can sync up
+
+* Start a Quorum Network, named it "mynetwork", consisting of "Node1,Node2,Node3"
+* Send some transactions to create blocks in network "mynetwork" and capture the latest block height as "latestBlockHeight"
+* Add "Node4" and join the network "mynetwork" as "nonvalidator"
+
+## A new node can sync up with the network but is not allowed to seal blocks
+
+  Tags: post-condition/datadir-cleanup, post-condition/network-cleanup, non-validator
+
+New node after being added to the network as non-validator node must not be able to seal new blocks
+
+* Deploy a simple smart contract from "Node4", verify it gets mined
+* Deploy a simple smart contract from "Node1", verify it gets mined
+* Record the current block number, named it as "blockHeightAfterContractsAreMinted"
+* Wait for node "Node4" to catch up to "blockHeightAfterContractsAreMinted"
+* Verify node "Node4" has the block height greater or equal to "blockHeightAfterContractsAreMinted"
```

Comment:
why we need this line if we already have the `Wait for node "Node4" to catch up to "blockHeightAfterContractsAreMinted"` that should timeout if I doesn't catch up?

Reply:
same applies in the other specs

Reply:
Done. Pushed a commit remove it